frontend/pages/add_salary.vue

```diff
@@ -621,9 +621,14 @@ export default {
             behavior: 'smooth',
             block: 'center',
           })
-        } else {
-          this.$store.dispatch('postRating', this.newRating)
-          this.$router.push('/')
+        }else if(this.newRating.salary < 10000){
+            if(confirm("Are you sure of the amount of salary you have entered? ")){
+              this.$store.dispatch('postRating', this.newRating)
+              this.$router.push('/')
+            }
+          } else {
```

Comment:
LGTM, but make sure to have a confirmation from a frontend here like @Asam237 !
Also, it could be great to have those lines formatted maybe 🤔 ?

Reply:
Thanks. I will contact @Asam237 for confirmation. 
For the code formatting will format it.